src/entities/hostile/Hostile.java

```diff
@@ -11,9 +11,7 @@
 public abstract class Hostile extends Entity{
 	
 	public int agro; //Bestimmt, wie sehr der Gegner den Spieler wahrgenommen hat <nicht implementiert>
-	public Entity target; //Zeigt auf wen der Gegner aggresiv ist <nicht implementiert>
-	
-	public Image image; //Image of the Item

```

Comment:
this fucking broke me.
3h wasted time.
Because image was double declared in hostile and in entity